answers.md

```diff
@@ -6,45 +6,46 @@ Some of these issues would not be WCAG 2.1 failures. But, they could present iss
+The tab order is a bit random, when you're tabbing through the list of links on the right-hand side of the page. This is unpredictable, and a bit disorienting. Whilst it's not a huge issue in this scenario, if the links were further apart on the page, or spread out more, it could cause issues for users who use a screen magnifier or browser zoom. They might not be able to tell where focus is gone, and have to hunt from it, or might have had their screen view dragged across the page as it's following focus (depending on how they've configured their assistive technologies).
 
 - Link opens in new tab
-Opening a link in a new tab can be appropriate if you tell the user in advance that it's going to happen. This can be done with either with text in the link description or with an icon that has alt text. In this scenario, neither of these techniques have been used. This is an issue because the user then can't use the back button on the keyboard to navigate to where they were, as a new tab history has been opened.
+Opening a link in a new tab can be appropriate if you tell the user in advance that it's going to happen. This can be done with either with content in the link text or with an icon that has appropriate alternative text. In this case, neither of these techniques have been used. This is an issue because the user may not be aware the the new tab or windows has opened. They then can't use the back button on the keyboard to navigate to where they were, as a new tab history has been opened.
 
 - Undescriptive link text
-Sometimes, screen reader users view links out of context. Some screen readers can pull all of the links out of a page and put them into one list, that is quicker to skim through. If link text doesn't make sense out of context, it would be unusable when viewed this way. The 'More' link on the start page doesn't make any sense out of context - we can infer it means more about geese by seeing the visual link with the section header, but this isn't sufficient.
+Screen reader users can view links out of context through a links menu. This can be quicker to skim through. If link text doesn't make sense out of context, it would be unusable when viewed this way. The 'More' link on the start page doesn't make any sense out of context - we can infer it means more about geese by seeing that it is in a section titled 'Information about Geese'. This isn't a failure against WCAG 2.1 Level AA criteria but is really useful to do.
```

Comment:
I have a question about the last part of this - would it be considered a WCAG failure against Link Purpose in Context - 2.4.4: https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html?

Reply:
The heading provides the context for the user and the link text has meaning within that context. This could be argued, I guess, as it is not clear that 'programmatically determined' includes the heading above the list.

Might better to say it 'isn't necessarily a failure'.

Reply:
Ah! I'd not thought of it like that. I was thinking of the thing you can do with VoiceOver or JAWS, where you pull all the links out of context, so you just get the link text in one long list - similar to how you can do with headings. In that use case, you'd not have the heading context, and would just get 'More' which wouldn't make sense, and so I'd always thought that it's a very clear WCAG failure - interested in other opinions on it though!

Reply:
This is an interesting detail I only learned recently.
2.4.4 AA: allows for the context to be outside the link text 
>"in the same sentence, paragraph, list item, or table cell as the link, or in the table header cell for a link in a data table, because these are directly associated with the link itself"
2.4.9 AAA: Requires the context to be in the link text.

But 2.4.4 also says 
>"Whenever possible, provide link text that identifies the purpose of the link without needing additional context"
so despite the technicality I would still highlight when testing for AA compliance.

I've also made a printable Word document of the answers with edits to the wording. My version is below, where I try to hide the nuance and promote good practice. In my view in this exercise we are encouraging people to be more accessibility not test/meet WCAG.

> "**Undescriptive link text**: Often screen reader users read links out of context. They can call up a list of all of the links on page to make them quicker to find and use. If link text does not make sense out of context, it would be unusable when read this way. The 'More' link doesn't make any sense on its own. It can be inferred or guessed, but that isn't sufficient. "

Reply:
@jbuller I agree that the goal should be a good experience for people. However, it is important to distinguish between what is good practice and what is a compliance failure. Whether we like it or not, some people will be concerned with compliance only and contractually obliged to meet the guidelines only.

Reply:
I had completely missed that nuance - thank you for sharing! The new suggested text conveys the importance and reasoning for why it's a good thing to do, but is also now clear about it's WCAG status, so I'm happy for this to go in - are you, James, given you've input too?

Should we also update the intro text a little, to make the WCAG thing a bit clearer? I might suggest an update that in a second PR or an issue, and mention you both in it, in case you're interested in inputting on that too.

Reply:
Might not be a bad thing to flag up the distinction between compliance and best practice.

I am happy to look at anything else that pops up :)

Reply:
Thank you for this, and for the offer of looking at future stuff - I'll merge it in now :)